webapp/src/components/backstage/settings.tsx

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {FC, useEffect, useState} from 'react';
+import {getProfiles, searchProfiles} from 'mattermost-redux/actions/users';
+import {getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
+import {GlobalState} from 'mattermost-redux/types/store';
+import {useDispatch, useSelector} from 'react-redux';
+
+import styled from 'styled-components';
+
+import {actionSetGlobalSettings} from 'src/actions';
+import {fetchGlobalSettings, setGlobalSettings} from 'src/client';
+import {PROFILE_CHUNK_SIZE} from 'src/constants';
+import {globalSettings} from 'src/selectors';
+import {GlobalSettings} from 'src/types/settings';
+import ConfirmModal from '../widgets/confirmation_modal';
+
+import Profile from '../profile/profile';
+
+import {useCanCreatePlaybooks} from 'src/hooks';
+
+import InviteUsersSelector from './automation/invite_users_selector';
+import {AutomationHeader, AutomationTitle, SelectorWrapper} from './automation/styles';
+import {Toggle} from './automation/toggle';
+import {BackstageHeader} from './styles';
+
+const SettingsContainer = styled.div`
+    margin: 0 auto;
+    max-width: 700px;
+`;
+
+const NoPermissionsTitle = styled.div`
+    font-weight: 600;
+    margin-bottom: 1rem;
+`;
+
+const NoPermissionsUsers = styled.div`
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+`;
+
+const NoPermissionsUserEntry = styled.div`
+    margin: 5px;
+`;
+
+interface PlaybookCreatorsProps {
+    settings: GlobalSettings
+    onChange: (newsettings: GlobalSettings) => void
+}
+
+const PlaybookCreators: FC<PlaybookCreatorsProps> = (props: PlaybookCreatorsProps) => {
+    const dispatch = useDispatch();
+    const [enabled, setEnabled] = useState<boolean>(Boolean(props.settings.playbook_editors_user_ids) && props.settings.playbook_editors_user_ids?.length !== 0);
```

Comment:
Can't this be simplified as follows (not sure, have not tested)?
```suggestion
    const [enabled, setEnabled] = useState<boolean>(Boolean(props.settings.playbook_editors_user_ids?.length !== 0));
```

Reply:
Nope, because `undefined !== 0` evaluates to `true`

Reply:
Urgh, that's true!